sadu-datasource/src/main/java/de/chojo/sadu/datasource/DataSourceCreator.java

```diff
@@ -69,7 +69,7 @@ private void loadDriverClass() {
         try {
             Class.forName(builder.driverClass());
         } catch (ClassNotFoundException e) {
-            throw new RuntimeException("Could not load driver class. Is class in class path? Use #setDriverClass when using relocation.", e);
+            throw new DriverClassNotFoundException("Could not load driver class. Is class in class path? Use #setDriverClass when using relocation.");
```

Comment:
Please pass the source exception into the new exception.
Currently the information which class we tried to load would be lost, since this information is only included in the original exception.

Reply:
done :)